src/api/builds.ts

```diff
@@ -107,17 +108,16 @@ export async function triggerBuild(
       body.AssetVersions = assetVersions;
     }
 
-    const resp = await client.post(`/Services/${serviceSid}/Builds`, {
-      // @ts-ignore
-      json: false,
+    const resp = await client.post(`Services/${serviceSid}/Builds`, {
+      responseType: 'json',
       headers: {
         'Content-Type': 'application/x-www-form-urlencoded',
       },
       body: querystring.stringify(body),
```

Comment:
I think there was something specific about how this worked with build in the initial version that I wrote up for this a while ago because of the way arrays are handled. But might be worth revisiting.